website/content/api-docs/secret/pki.mdx

```diff
@@ -799,544 +808,506 @@ $ curl \
 
-~> **Note**: ECDSA and Ed25519 issuers do not follow configuration of the
-   `signature_bits` value; only RSA issuers will change signature types
-   based on this parameter.
+#### Sample Request
 
-- `key_usage` `(list: ["DigitalSignature", "KeyAgreement", "KeyEncipherment"])` –
-  Specifies the allowed key usage constraint on issued certificates. Valid
-  values can be found at https://golang.org/pkg/crypto/x509/#KeyUsage - simply
-  drop the `KeyUsage` part of the value. Values are not case-sensitive. To
-  specify no key usage constraints, set this to an empty list. See
-  [RFC 5280 Section 4.2.1.3](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.3)
-  for more information about the Key Usage field.
+```shell-session
+$ curl \
+    http://127.0.0.1:8200/v1/pki/ca_chain
+```
 
-- `ext_key_usage` `(list: [])` –
-  Specifies the allowed extended key usage constraint on issued certificates. Valid
-  values can be found at https://golang.org/pkg/crypto/x509/#ExtKeyUsage - simply
-  drop the `ExtKeyUsage` part of the value. Values are not case-sensitive. To
-  specify no key usage constraints, set this to an empty list. See
-  [RFC 5280 Section 4.2.1.12](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.12)
-  for information about the Extended Key Usage field.
+#### Sample Response
 
-- `ext_key_usage_oids` `(string: "")` - A comma-separated string or list of extended
-  key usage oids. Useful for adding EKUs not supported by the Go standard library.
+```text
+<PEM-encoded certificate chain>
+```
 
-- `use_csr_common_name` `(bool: true)` – When used with the CSR signing
-  endpoint, the common name in the CSR will be used instead of taken from the
-  JSON data. This does not include any requested SANs in the CSR; use
-  `use_csr_sans` for that.
+<a name="read-crl"></a>
 
-- `use_csr_sans` `(bool: true)` – When used with the CSR signing endpoint, the
-  subject alternate names in the CSR will be used instead of taken from the JSON
-  data. This does not include the common name in the CSR; use
-  `use_csr_common_name` for that.
+### Read Issuer CRL
 
-- `ou` `(string: "")` – Specifies the OU (OrganizationalUnit) values in the
-  subject field of issued certificates. This is a comma-separated string or
-  JSON array.
+This endpoint retrieves the specified issuer's CRL.
 
-- `organization` `(string: "")` – Specifies the O (Organization) values in the
-  subject field of issued certificates. This is a comma-separated string or
-  JSON array.
+Note that the response differs between the older `/pki/cert/crl` path and
+the newer `/pki/issuer/:issuer_ref/crl` path; the latter correctly places the
+PEM-encoded CRL in the `crl` field whereas the former incorrectly places it
+in the `certificate` field.
 
-- `country` `(string: "")` – Specifies the C (Country) values in the
-  subject field of issued certificates. This is a comma-separated string or
-  JSON array.
+These are unauthenticated endpoints.
```

Comment:
Should we add a note here saying as of Vault 1.11 CRL are now in v2 format or something?

Reply:
Yeah I like it.